Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```
@@ -140,7 +140,7 @@
   name: cobra
   parent: [ BaseWeaponPistol, BaseSyndicateContraband ]
   id: WeaponPistolCobra
-  description: A rugged, robust operator handgun with inbuilt silencer. Uses .25 caseless ammo.
+  description: Integrally suppressed semi-automatic pistol used by the Syndicate, it fires low caliber subsonic rounds. Favored by any agent who likes to keep things quiet and leave no evidence behind. Feeds from .25 pistol magazines.
```

Description needs to state that it fires caseless rounds or else you might troll players. Low caliber is partially irrelevant here.

```suggestion
  description: Integrally suppressed semi-automatic pistol used by the Syndicate, firing caseless subsonic rounds. Favored by any agent who likes to keep things quiet and leave no evidence behind. Feeds from .25 pistol magazines.
```